### PR TITLE
Inception can change independently of the RRs/hash

### PR DESCRIPTION
**Pull request by jablko**
so update the oracle regardless.

### CODE REVIEW

**Comment by jablko** [comment]
> Can you add a test that demonstrates this?

Good idea, added.

File: test/TestDNSSEC.js [hunk 1]
```diff
@@ -110,6 +110,30 @@ async function verifyFailedSubmission(instance, data, sig, proof) {
 }
 
 contract('DNSSEC', function(accounts) {
+  before(async () => {
```
Comment (Arachnid) [inline test/TestDNSSEC.js]: What are these changes for ?

Comment (jablko) [reply test/TestDNSSEC.js]: The new test fails without them, depending on whether the other tests are run and how quickly or slowly they execute. It depends on the state of the inception value in the oracle.

Comment (Arachnid) [reply test/TestDNSSEC.js]: The existing tests are written sequentially, with each building on the last. It's not ideal, but it's the way Truffle works. Can you add the new test in the same fashion?

Comment (jablko) [reply test/TestDNSSEC.js]: Done, however I think isolating the tests from each other is a worthwhile improvement. Can I please move that commit to its own pull request?

Comment (Arachnid) [reply test/TestDNSSEC.js]: Sure, that seems reasonable.

**Pull request merged**